examples/rn-todo-list/frontend/sync.config.js

```diff
@@ -0,0 +1,24 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+export const SYNC_CONFIG = {
+  // Set `enabled` to `true` to enable sync.
+  enabled: true,
+  // Add your Atlas App ID here if sync is enabled.
+  appId: 'YOUR_APP_ID',
+};
```

Comment:
Since this app uses two versions (a local-only and a sync one), I did not refactor this part into the convention used by the other examples (i.e. having an `ATLAS_APP_ID` variable exported in `app/atlas-app-services/config.ts`. Feel free to say if you'd rather see this following the same convention as the other apps.

Note:
Previous default for `enabled` was `false`.

Reply:
With `enabled` set to `true`, the app will fail early which can guide the user to try Device Sync - smart move :smile:

I don't see an issue that one example deviate from the other example so let's keep it as it is.